modules/test/playwright/pages/account-admin-web/AccountUsersPage.ts

```diff
@@ -37,11 +40,22 @@ export class AccountUsersPage {
 		this.assignUserMenuItem = page.getByRole('menuitem', {
 			name: 'Assign Users',
 		});
+		this.clearButton = page.getByRole('button', {name: 'Clear'});
 		this.deactivateButton = page
 			.getByRole('button', {
 				name: 'Deactivate',
 			})
 			.or(page.getByRole('link', {name: 'Deactivate'}));
+		this.filterButton = page.getByRole('button', {
+			exact: true,
+			name: 'Filter',
+		});
+		this.filterMenuItem = (option: string) => {
+			return page.getByRole('menuitem', {
+				exact: true,
+				name: option,
+			});
+		};
```

Comment:
You can reach the filter and set it using the already existing filters inside the `usersTable`